lib/puppet/pops/issues.rb

```diff
@@ -58,7 +58,8 @@ def format(hash ={})
   # in the issue's message producing block.
   # @api private
   #
-  class MessageData
+  class MessageData < Puppet::Pops::LabelProvider
```

Comment:
i.e. this class must have the mehods, and they must then call the same method on the provider.